docs/source/contributing/local-deployment.rst

```diff
@@ -169,32 +163,28 @@ Launching
 
       skaffold run -p dev,arm64
 
+.. tip::
+   If you need to re-run `skaffold run` for whatever reason, don't forget to use `skaffold delete` to reset the state beforehand (or reset your environment by running the `k3-create.sh` script again).
+
 .. tip::
    When re-launching the orchestrator and the backend, you can speed up the processing by avoiding the update of the chart dependencies using the profile ``nodeps``.
 
    .. code-block:: bash
 
       skaffold run -p nodeps
 
-* Deploy the frontend. You can use two methods (described below)
-
-  a. Local server: Execute the following command:
+* Deploy the frontend
 
-    .. code-block:: bash
-
-      cd substra-frontend
-      API_URL=http://substra-backend.org-1.com npm run dev
-
-  b. Docker:
-
-     .. code-block:: bash
+   .. code-block:: bash
 
-      docker build -f docker/substra-frontend/Dockerfile --target dev -t substra-frontend .
-      docker run -it --rm -p 3000:3000 --name DOCKER_FRONTEND_CONTAINER_NAME -e API_URL=http://substra-backend.org-1.com -v ${PWD}/src:/workspace/src substra-frontend
+        cd substra-frontend
+        docker build -f docker/substra-frontend/Dockerfile --target dev -t substra-frontend .
+        docker run -it --rm -p 3000:3000 -e API_URL=http://substra-backend.org-1.com -v ${PWD}/src:/workspace/src substra-frontend
 
-     | with ``DOCKER_FRONTEND_CONTAINER_NAME`` the name of the frontend container that will be used for the rest of the operations.
+  You can access the frontend at http://substra-frontend.org-1.com:3000/. The dev credentials are:
 
-  * In both case, you can access the frontend at http://substra-frontend.org-1.com:3000/.
+    * login: org-1
+    * password: p@sswr0d44
```

Comment:
the @ in the password creates an mailto hyperlink :/. Do we have a way to remove the hyperlink?
<img width="265" alt="image" src="https://user-images.githubusercontent.com/5354301/215805667-cc3851b6-06e5-4a2b-ad6b-364d92f94146.png">

Reply:
Another suggestion to make the login/pwd stand out (I haven't tested this)
```suggestion
    * login: ``org-1``
    * password: ``p@sswr0d44``
```

Reply:
well spotted 👍